### DISCUSSION TITLE
Callback stats - What does this sensor mean?

### DISCUSSION THREAD

**Discussion by man55** [kellerza/sunsynk#337] [Q&A]
Callback stats - What does this sensor mean?
I have not found a single mention anywhere, any description of what this sensor shows. 
Could you please explain? Thank you.

**Comment by kellerza** [comment]
It shows read timeouts. If it increases/stays high you probably have some issue, could try reducing the read intervals

**Reply by man55** [reply]
Thank you. And how much is a high? 10, 100, 1000?
My sensor value fluctuates around 40.
At the same time, RS485 timeout is always zero.
And what is the difference between these two sensors?

It would also be nice to somehow change Callback stats sensor so that it doesn't spam the logbook with every change.

p.s.
Oh, I just saw that there are many more attributes there. What does it all mean?
```
period: 120s
long_calls: 24/42
long_calls_avg: 3.88s
slips: 78/120
slips_avg: 2.15s
busy: 78
```

**Reply by kellerza** [reply]
Can add a setting to disable it if it's an issue. It probably makes little sense without understanding how the internal callbacks work.

**Reply by man55** [reply]
I just wanted to understand if there is a problem here or not. And if there is, at what stage of communication does it occur: at the stage of a long wireless connection between HomeAssistant and raspberry with mbusd, or between mbusd and the rs485 converter or inverter?
I tried one rs485 converter which I threw in the trash, because the log was full of errors reading sensors.
Maybe another converter is not very good either, there are too many of them and they are built on different chips, and there are also many fakes. And it is difficult to determine without trying.
I read the Adaptors & Wiring page, but the situation on the local market is a little different.
